bundler/spec/commands/install_spec.rb

```diff
@@ -888,16 +892,16 @@ def run
   context "with missing platform specific gems in lockfile" do
     before do
       build_repo4 do
-        build_gem "racc", "1.5.2"
+        build_gem "racca", "1.5.2"
```

Comment:
Can we explain this? Commit message mentions a conflict with the default gem? Is this a test only issue, or could that happen in real life?

Reply:
good question, I don't 100% remember